packages-native/popup-menu/src/PopupMenu.tsx

```diff
@@ -12,13 +13,10 @@ import {
 } from "react-native";
 import { ActionValue } from "mendix";
 import Menu, { MenuDivider, MenuItem } from "react-native-material-menu";
-import deepmerge from "deepmerge";
+import flatten = StyleSheet.flatten;
```

Comment:
I would just skip this `import` and write `StyleSheet.flatten` in one place where you need it. Imho would make code more readable.